src/components/InputWithLabel.jsx

```diff
@@ -23,3 +28,9 @@ export default function InputWithLabel({
     </>
   );
 }
+
+InputWithLabel.proptype = {
+  children: PropTypes.string.isRequired,
```

Comment:
I think you may want to revisit your proptype here for `children`. Are you confident that it will always be a string? Feel free to cross-reference with the docs here to see what other prop types you have at your disposal. https://github.com/facebook/prop-types